From 9aaf78f31a3370fcfa4cbe111406a9a6e8b79c1d Mon Sep 17 00:00:00 2001
From: Michael Goffioul <michael.goffioul@lincor.com>
Date: Fri, 4 Oct 2019 20:47:05 -0400
Subject: [PATCH 3/7] Setup /dev/ion permissions/ownership

---
 init.x86.rc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/init.x86.rc b/init.x86.rc
index b36a722..241bb1f 100644
--- a/init.x86.rc
+++ b/init.x86.rc
@@ -17,6 +17,10 @@ on fs
 on post-fs
     exec u:r:init:s0 -- /system/bin/logwrapper /system/bin/sh /system/etc/init.sh
 
+on post-fs
+    chmod 0664 /dev/ion
+    chown system system /dev/ion
+
 on post-fs-data
     # Create the directories used by the Wireless subsystem
     mkdir /data/vendor/wifi 0771 wifi wifi
-- 
2.21.0

