From 4e1c4910014019726c55ca1a24616a8430a334bc Mon Sep 17 00:00:00 2001
From: Michael Goffioul <michael.goffioul@lincor.com>
Date: Fri, 4 Oct 2019 21:01:58 -0400
Subject: [PATCH 6/7] Use XML files for audio configuration

---
 BoardConfig.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index 453b513..1e6a7bf 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -92,3 +92,4 @@ BOARD_SEPOLICY_DIRS += device/generic/common/sepolicy \
 
 BOARD_BUILD_SYSTEM_ROOT_IMAGE:= true
 BUILD_BROKEN_PHONY_TARGETS := true
+USE_XML_AUDIO_POLICY_CONF := 1
-- 
2.21.0

