From 8b565b81a1225c2267eedbdd0434fcf329f426e5 Mon Sep 17 00:00:00 2001
From: Jon West <electrikjesus@gmail.com>
Date: Mon, 4 Mar 2019 21:04:20 -0500
Subject: [PATCH 4/8] link /lib/modules as well

---
 initrd/init | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/initrd/init b/initrd/init
index 746cffd..21b3c9a 100755
--- a/initrd/init
+++ b/initrd/init
@@ -169,6 +169,8 @@ cd /android
 ln -s mnt/$SRC /src
 ln -s android/system /
 ln -s ../system/lib/firmware ../system/lib/modules /lib
+mkdir -p /lib/modules
+ln -s /system/lib/modules /lib/modules/`uname -r`
 
 mkdir /apex
 if [ -d system/apex/com.android.runtime.debug ]; then
-- 
2.17.1

