From 1fe0bb652af6c0ca91a0ecc9ca1fa34af421510a Mon Sep 17 00:00:00 2001
From: Jon West <electrikjesus@gmail.com>
Date: Tue, 5 Mar 2019 15:42:35 -0500
Subject: [PATCH 5/8] Add back nvme device mounting

Conflicts:
	initrd/init
---
 initrd/init | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/initrd/init b/initrd/init
index 21b3c9a..fbf2a19 100755
--- a/initrd/init
+++ b/initrd/init
@@ -157,7 +157,7 @@ done
 
 cd /
 while :; do
-	for device in ${ROOT:-/dev/[hmnsv][dmrv][0-9a-z]*}; do
+	for device in ${ROOT:-/dev/[hmnsv][dmrv][0-9a-z]*} ${ROOT:-/dev/xvd[0-9a-z]*} ${ROOT:-/dev/nvme0n[0-9]p[0-9a-z]*}; do
 		check_root $device && break 2
 		mountpoint -q /mnt && umount /mnt
 	done
-- 
2.17.1

