From 81a9735c817f7870d8839c4b8e4e0cf0161f1bd6 Mon Sep 17 00:00:00 2001
From: Michael Goffioul <michael.goffioul@lincor.com>
Date: Fri, 4 Oct 2019 20:19:48 -0400
Subject: [PATCH 2/2] Ignore build subdir

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index dce4a8149d2c..c2e9a95c12db 100644
--- a/.gitignore
+++ b/.gitignore
@@ -134,3 +134,4 @@ all.config
 *.kdev4
 
 configs
+build
-- 
2.21.0

